lib/plugin.dart

```diff
@@ -153,9 +153,15 @@ class AtomDartPackage extends AtomPackage {
         'type': 'boolean',
         'default': false
       },
-      'showUnnamedLibraryWarning': {
+      'filterUnnamedLibraryWarnings': {
```

Comment:
While slightly inaccurate, this setting was named `showUnnamed...` in order for it to display after the other settings in the preferences page. They're sorted (I believe) by pref id. `filter` will sort before everything else, and these aren't the primary prefs for the app.

We can play with some id prefixes in order to group settings; let's keep it as you have for now.